lib/active_admin/inputs/filters/date_range_input.rb

```diff
@@ -39,7 +39,7 @@ def input_html_options_for(input_name, placeholder)
                             nil
                           end
           { placeholder: placeholder,
-            value: current_value ? current_value.strftime("%Y-%m-%d") : "" }.merge(input_html_options)
+            value: current_value ? current_value.to_s : "" }.merge(input_html_options)
```

Comment:
If this is correct, then it should be the same as

```suggestion
            value: current_value.to_s }.merge(input_html_options)
```